en/02_Developer_Guides/15_Customising_the_Admin_Interface/07_jQuery_Entwine.md

```diff
@@ -383,6 +375,9 @@ $('div').entwine({
 
 With the above entwine declarations, calling
 
+> [!NOTE]
+> Note that trying to call `$('div').bar();` would throw an uncaught `TypeError` saying something like "$(...).bar is not a function", because the `bar()` function was defined in a namespace, but we are trying to call that function from *outside* of that namespace.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.